xsamtools/cram.py

```diff
@@ -23,6 +23,108 @@
 log = logging.getLogger(__name__)
 
 
+def next_int(fh):
```

Comment:
please add parameter and return type hints. This should be applied throughout this PR.